tests/integration/test_kfp_functional.py

```diff
@@ -127,9 +126,9 @@ async def test_build_and_deploy(ops_test: OpsTest, request, lightkube_client):
 async def test_upload_pipeline(kfp_client):
     """Upload a pipeline from a YAML file and assert its presence."""
     # Upload a pipeline and get the server response
-    pipeline_name = f"test-pipeline-sdk-{KFP_SDK_VERSION}"
+    pipeline_name = f"test-pipeline-sdk"
```

Comment:
we still have the `-sdk` suffix in some places. I think it'd be better to remove it completely or just keep v2 hardcoded.